src/scss/_themes.scss

```diff
@@ -23,12 +23,38 @@
 	}
 }
 
+/// Portrait card theme styles
+///
+/// @param {String} $modifier [portrait] - the modifier classname you want to use
+@mixin oCardThemePortrait($modifier: 'portrait') {
+
+	&--#{$modifier} {
+		flex-direction: column;
+		flex-wrap: nowrap;
+		justify-content: flex-start;
+	}
+
+	// Normalise the styles for an image in a portrait card
+	&--#{$modifier} &__image {
```

Comment:
I would even suggest making the modifiers fixed, just to be consistent and only letting the base class be editable